iele-gas.md

```diff
@@ -807,18 +807,18 @@ Note: These are all functions as the operator `#compute` has already loaded all
  // ---------------------------------------------
     rule #allBut64th(N) => N -Int (N /Int 64)
 
-    syntax Int ::= G0 ( Schedule , WordStack , Ints ) [function, klabel(G0create)]
-                 | G0 ( Schedule , String , Ints )    [function, klabel(G0call)]
-                 | G0 ( Schedule , WordStack , Bool ) [function, klabel(G0aux)]
+    syntax Int ::= G0 ( Schedule , Bytes , Ints )  [function, klabel(G0create)]
+                 | G0 ( Schedule , String , Ints ) [function, klabel(G0call)]
+                 | G0 ( Schedule , Bytes , Bool )  [function, klabel(G0aux)]
  // ---------------------------------------------------------------------------
```

Comment:
formatting

Reply:
What are you looking for here, lining up the commas/parens?

Reply:
Ah nevermind, I read it wrong